Makefile

```diff
@@ -94,7 +94,16 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
-
+.PHONY: install-git-hooks
+## Creates useful git hooks (e.g. pre-push go-format-code check)
+install-git-hooks:
+	ln -sfv ../../.pre-push .git/hooks/pre-push
```

Comment:
@hectorj2f it is actually `check-go-format` what's executed ;) The one you mentioned does format the code which is not what I want to do automatically.